plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/analyzeRefinedCallShape.kt

```diff
@@ -38,44 +32,6 @@ fun KotlinTypeFacade.analyzeRefinedCallShape(call: FirFunctionCall, reporter: In
 
     val newSchema: PluginDataFrameSchema = call.interpreterName(session)?.let { name ->
         when (name) {
-            "toDataFrameDsl" -> {
-                val list = call.argumentList as FirResolvedArgumentList
-                val lambda = (list.arguments.singleOrNull() as? FirAnonymousFunctionExpression)?.anonymousFunction
-                val statements = lambda?.body?.statements
-                if (statements != null) {
-                    val receiver = CreateDataFrameDslImplApproximation()
-                    statements.filterIsInstance<FirFunctionCall>().forEach {
-                        val schemaProcessor = it.loadInterpreter() ?: return@forEach
-                        interpret(
-                            it,
-                            schemaProcessor,
-                            mapOf("dsl" to Interpreter.Success(receiver), "call" to Interpreter.Success(call)),
-                            reporter
-                        )
-                    }
-                    PluginDataFrameSchema(receiver.columns)
-                } else {
-                    PluginDataFrameSchema(emptyList())
-                }
-            }
-            "toDataFrame" -> {
-                val list = call.argumentList as FirResolvedArgumentList
-                val argument = list.mapping.entries.firstOrNull { it.value.name == Name.identifier("maxDepth") }?.key
-                val maxDepth = when (argument) {
-                    null -> 0
-                    is FirLiteralExpression -> (argument.value as Number).toInt()
-                    else -> null
-                }
-                if (maxDepth != null) {
-                    toDataFrame(maxDepth, call, TraverseConfiguration())
-                } else {
-                    PluginDataFrameSchema(emptyList())
-                }
-            }
-            "toDataFrameDefault" -> {
-                val maxDepth = 0
-                toDataFrame(maxDepth, call, TraverseConfiguration())
-            }
             "Aggregate" -> {
```

Comment:
sorry, again better to have CONSTANTS, believe that these names are repeated in codebase

Reply:
Answered in other PR :)